contrib/mesos/pkg/scheduler/plugin.go

```diff
@@ -52,6 +52,11 @@ const (
 	pluginRecoveryDelay = 100 * time.Millisecond // delay after scheduler plugin crashes, before we resume scheduling
 )
 
+const (
+	FailedScheduling = "FailedScheduling"
+	Scheduled        = "Scheduled"
+)
+
 // scheduler abstraction to allow for easier unit testing
 type schedulerInterface interface {
```

Comment:
I'm really not a fan of this naming convention using `-Interface`. 
Are we doing this throughout k8sm? Is the rest of k8s doing this?